deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-ocm-agent-alerts
+    role: alert-rules 
+  name: sre-ocm-agent-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-ocm-agent-alerts
+    rules:
+    - alert: KubePersistentVolumeFillingUp
```

Maybe something like 

```suggestion
    - alert: LoggingVolumeFillingUpNotificationSRE
```

---

Fixed